concepts/00001-enarx-sprint-framework/README.md

```diff
@@ -0,0 +1,158 @@
+# Agile Sprint Process for Enarx
+- Authors: [Mark Bestavros](mbestavr@redhat.com) -- email is optional
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-05-06 (date you submit your PR)
+- Status Note: Concept brainstormed, awaiting implementation
+- Supersedes: N/A
+- Start Date: 2020-05-01 (date you started working on this idea)
+- Tags: process, sprint, concept
+
+## Summary
+
+Enarx has been using its daily standup and board organization for a while. It's worked for us as a small team, but as we grow, we anticipate it becoming unsustainable. It also doesn't do a good job capturing the current thrust of our work. This RFC proposes a replacement process, designed around the agile sprint methodology.
+
+## Motivation
+
+The Enarx project needs to organize its work better, and in a more sustainable way than before. We need to do a better job planning for medium-term work and goals, and in a way that scales more effectively than our prior process.
+
+## Tutorial
+
+Before we start thinking about how our sprints will work, we should lay the groundwork for how our backlog and sprint boards should be organized.
+
+### Triage
+
+By default, we shouldn't do anything when an issue is first submitted. The lack of metadata will indicate it needs to be triaged. (If desired, we could post a quick welcome comment if they're not part of the core team, but that's optional.)
+
+Every standup, we'll look at all issues that are not part of a project and not labeled. This ensures that new issues always get a human response within 24 working hours, and that all issues are organized properly into the backlog -- or, if needed, promoted to the current sprint. It also allows the team to discuss all new issues as they come up.
+
+### Backlog
+
+As our project scales, it’s easy to imagine a single unified backlog becoming incredibly cumbersome and hard to manage. Instead, we should adapt our current approach to area-specific boards, which effectively divides our backlog into categories (SGX, SEV,  infrastructure, etc.).
```

Comment:
My first reaction as I read this paragraph was to agree. However, I think there might be some value to consolidating our boards into a single board. The primary reason is that when we're sprinting, I'd argue that the single most important unit of work is our active sprint (comprised of issues that should arguably sum to a reasonable amount of work within a workweek per engineer). By having just one board, we can capture the contents of the sprint with one column on one board, rather than having this fragmented across component-level boards. I think we can also accomplish this with GitHub milestones.

In any case, I don't think smaller boards buy us anything. Even if the rate at which we accumulate work vastly outpaces the work completed per sprint, I think we'd benefit more from disciplined grooming of the backlog and decomposition of tasks than fragmenting the backlog into component-sized boards. We will need to advocate ruthlessly for understanding the priority of individual issues and how they fit in with our roadmap moving forward. Regardless of the number of boards, the size of the backlog is the size of the backlog, regardless of if it's divided by 1 board or N-boards.

Reply:
I just checked, and we are able to filter cards by labels while on a board, so this could help us stay organized if we do decide to have just one backlog instead of multiple backlogs.

Reply:
A "Sprint board" is going to be important.

Reply:
I am noting consensus on a single sprint board.

Reply:
Resolving to simplify older comments.